section1/exercises/variables.rb

```diff
@@ -1,59 +1,68 @@
 # In the below exercises, write code that achieves
 # the desired result. To check your work, run this
-# file by entering the following command in your terminal: 
+# file by entering the following command in your terminal:
 # `ruby section1/exercises/variables.rb`
 
 # Example: Write code that saves your name to a variable and
 # prints what that variable holds to the terminal:
-name = "Harry Potter"
-p name
+name = "Matt Holmes"
+puts name
 
 # Write code that saves the string 'Dobby' to a variable and
 # prints what that variable holds to the terminal:
 house_elf = "Dobby"
-# YOUR CODE HERE
+puts house_elf
 
 # Write code that saves the string 'Harry Potter must not return to Hogwarts!'
 # and prints what that variable holds to the terminal:
-# YOUR CODE HERE
+# instruction = "Harry Potter must not return to Hogwarts!"
+puts instruction
 
 # Write code that adds 2 to the `students` variable and
 # prints the result:
 students = 22
-# YOUR CODE HERE
+puts #{students + 2}
 p students
 
 # Write code that subracts 2 from the `students` variable and
 # prints the result:
-# YOUR CODE HERE
+# puts #{students - 2}
 p students
 
 
 # YOU DO:
-# Declare three variables, named `first_name`, `is_hungry` and `number_of_pets`. 
+# Declare three variables, named `first_name`, `is_hungry` and `number_of_pets`.
 # Store the appropriate data types in each.
 # print all three variables to the terminal.
+first_name = "Matt"
+is_hungry = True
+number_of_pets = 0
+puts "Is #{first_name} hungry?" is_hungry "." "#{first_name} has #{number_of_pets} pets."
```

Comment:
There is a break in your code here. Try running this file, follow the error and see if you can update for working code. Hint: read the string you are creating out loud, character by character. Is that what you were trying to create?